src/lib/autoinstall/widgets/storage/stripesize.rb

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) [2020] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "yast"
+require "y2storage"
+require "cwm/common_widgets"
+
+module Y2Autoinstallation
+  module Widgets
+    module Storage
+      # Widget to specify the granularity of the stripes
+      class Stripesize < CWM::IntField
+        # Constructor
+        def initialize
+          textdomain "autoinst"
+          super
+        end
+
+        # @macro seeAbstractWidget
+        def label
+          _("Stripe Size (in KB)")
```

Comment:
We should probably use the more correct "KiB" here.

Reply:
I will check whether is really interpreted as KB or KiB (in the past we used to misuse them).

Reply:
The [AutoYaST documentation](https://susedoc.github.io/doc-sle/master/single-html/SLES-autoyast/#ay-partition-configuration) says "KB".

If that is wrong, we should update the documentation too.

Reply:
Well, everything around AutoYaST (documentation, etc.) misuse the term. So let's not try to fix it here and simply use "KB" for consistency.